requirements.in

```diff
@@ -0,0 +1,5 @@
+pyyaml
```

Comment:
Where do these requirements come from?

Reply:
Good catch!  That's the old list from an older setup.py.  I've updated the pull request to use a single source of truth for requirements (stored in requirements.in, read from setup.py).